src/domains/community/write/components/WriteCommunityPostForm.jsx

```
@@ -12,9 +12,10 @@ import JobInputBox from '../../../../components/web/JobInputBox'
 import JobFormLine from '../../../job/common/components/JobFormLine'
 import useAuthStore from '../../../../store/login/useAuthStore'
 import WriteEditor from '../../../../components/common/WriteEditor'
-
-const WriteCommunityPostForm = () => {
+import useDraftStore from '../../../../store/mypage/useDraftStore'
+const WriteCommunityPostForm = ({ onSaveDraft }) => {
```

ESLint

[error] 'onSaveDraft' is missing in props validation

(react/prop-types)